packages/client/src/views/Performance/FieldAgentList.tsx

```diff
@@ -105,6 +108,27 @@ enum STATUS_OPTIONS {
   PENDING = 'pending'
 }
 
+const TableDiv = styled.div`
+  overflow: auto;
+`
+
+const NameAvatar = styled.div`
+  display: flex;
+  align-items: center;
+  img {
+    margin-right: 10px;
+  }
+`
+
+function getNameWithAvatar(userName: string, avatar: IAvatar) {
+  return (
+    <NameAvatar>
+      <AvatarSmall name={userName} avatar={avatar as IAvatar} />
```

Comment:
```suggestion
      <AvatarSmall name={userName} avatar={avatar} />
```
Should be sufficient, right?

Reply:
Yeah thanks